toonz/sources/toonz/cellselection.cpp

```diff
@@ -1849,6 +1906,96 @@ void TCellSelection::pasteKeyframesInto() {
 
 //-----------------------------------------------------------------------------
 
+void TCellSelection::duplicateFrame() {
+  if (!Preferences::instance()->isSyncLevelRenumberWithXsheetEnabled()) return;
```

Comment:
How about opening some warning popup notifying that the preference option should be changed to use this command ?